src/api/dependency-discovery/test/dependency.test.js

```diff
@@ -0,0 +1,17 @@
+const request = require('supertest');
+const dependencyList = require('../src/dependency-list');
+const { app } = require('../src');
+
+jest.mock('process', () => ({
+  cwd: () => 'src/api/dependency-discovery',
```

Comment:
Agree with @JerryHue , let's mock the file for the dependency list, so we can write tests against it.